spec.html

```diff
@@ -425,6 +425,11 @@ <h1>%TypedArray%.prototype.toSpliced ( _start_, _deleteCount_, ..._items_ )</h1>
                         1. Else if _relativeStart_ &lt; 0, let _actualStart_ be max(_len_ + _relativeStart_, 0).
                         1. Else, let _actualStart_ be min(_relativeStart_, _len_).
                         1. Let _insertCount_ be the number of elements in _items_.
```

Comment:
Would we want to do this even before we get the length, to take https://github.com/tc39/proposal-resizablearraybuffer into account?

Reply:
Nope actually, the precedence is to read-and-cache the length, then do conversions.